triedb/pathdb/disklayer.go

```diff
@@ -150,9 +150,9 @@ func (dl *diskLayer) markStale() {
 	dl.stale = true
 }
 
-// Node implements the layer interface, retrieving the trie node with the
-// provided node info. No error will be returned if the node is not found.
-func (dl *diskLayer) Node(owner common.Hash, path []byte, hash common.Hash) ([]byte, error) {
+type loggerFunc func(string, ...interface{})
+
+func (dl *diskLayer) NodeByLogger(owner common.Hash, path []byte, hash common.Hash, logger loggerFunc) ([]byte, error) {
```

Comment:
`logger` is so confusing here without any comments. I think we could remove it to keep it simplify.

Reply:
And if the logic of querying 128 difflayer again above is removed, it is not necessary here too.

Reply:
Updated.